analyses/survival-analysis/survival-analysis_immune.Rmd

```diff
@@ -138,7 +137,7 @@ stranded <- read_rds(file.path(data_dir, "pbta-gene-expression-rsem-fpkm-collaps
 # Process gene expression files to obtain PDL1 (aka CD274)
 cd274_expression <- bind_rows(
   extract_cd274_expression(polya, "polya"),
-  stranded_cd274 <- extract_cd274_expression(stranded, "stranded")
+  extract_cd274_expression(stranded, "stranded")
```

Comment:
I noticed this and couldn't help myself 😬

Reply:
too funny